CompetitionTask1/CompetitionTask1/Utilities/ModelJsonData.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2.Utilities
+{
+    public class ModelJsonData
+    {
+        
+    }
+    public class EducationJsonData
+    {
+        public List<EducationRecords> Create { get; set; }
+        public List<EducationRecords> Update1 { get; set; }
+        public List<EducationRecords> Update2 { get; set; }
```

Comment:
I'd suggest having your test data in different JSON files and you give them meaningful names instead of putting them all under this `EducationJsonData` class. Having them in this class is less flexible, e.g. if you want to add a second test case for creating a new education record, you have to change this class.

Reply:
As a reference, this was how I did it:
<img width="198" alt="image" src="https://github.com/VenkataSriA/InternshipMVP/assets/88355898/1f5a5a0f-1b7a-42aa-82c7-0a4383c52fb4">
<img width="486" alt="image" src="https://github.com/VenkataSriA/InternshipMVP/assets/88355898/7a1a75b0-7740-4a90-9338-8f81ee663bbf">
<img width="569" alt="image" src="https://github.com/VenkataSriA/InternshipMVP/assets/88355898/c4ea392a-9e62-4c33-a6f8-6da6c6f0bfee">